app/css/general.css

```diff
@@ -110,6 +114,14 @@ html,body {
 .button.current .icon{
 	-webkit-filter: invert(100%);
 }
+@keyframes fadeout {
+	from { opacity: 1; }
+	to   { opacity: 0; }
+}
+@keyframes fadein {
+	from { opacity: 0; }
+	to   { opacity: default; }
+}
```

Comment:
Small note I realized just now on Github line-note commenting. Comment on the exact line otherwise the automatic "___ commented on an outdated diff" won't autohide it.
![selection_005](https://cloud.githubusercontent.com/assets/4606838/9038325/732e4d98-39c1-11e5-85bf-8ecff8bd0464.png)